web/client/components/home/Home.jsx

```diff
@@ -23,20 +24,22 @@ class Home extends React.Component {
     };
 
     static defaultProps = {
-        icon: <Glyphicon glyph="home"/>
+        icon: <Glyphicon glyph="home"/>,
+        tooltipPosition: 'left'
     };
 
     render() {
+        const { tooltipPosition, ...restProps} = this.props;
         let tooltip = <Tooltip id="toolbar-home-button">{<Message msgId="gohome"/>}</Tooltip>;
         return (
-            <OverlayTrigger overlay={tooltip} placement="left">
+            <OverlayTrigger overlay={tooltip} placement={tooltipPosition}>
                 <Button
-                    {...this.props}
                     id="home-button"
                     className="square-button"
                     bsStyle="primary"
                     onClick={this.goHome}
                     tooltip={tooltip}
+                    {...restProps}
```

Comment:
With this change i'm able to override className and bsStyle, because i'm using this component in navigation of geostory. This change is needed to render home button in different way than default.

![Zrzut ekranu z 2019-12-11 14-34-23](https://user-images.githubusercontent.com/56867851/70625784-6e688c80-1c23-11ea-9048-738f13149ebb.png)